### PR TITLE
feat(chips): integrate component

### PR DESCRIPTION
**Pull request by mJarsater**
* Integrated chips components in chip page
* Used chip in form and outside of form
* Refactored styling of <main>
* Added utility class for flex-wrap
* Added reset styling for fieldset

### CODE REVIEW

File: src/app/pages/chips-page/chips-page.component.html [hunk 1]
```diff
@@ -1 +1,55 @@
-<h1>Chips</h1>
+<h1>Start testing Chips by exploring the actions below</h1>
+<p>
+  Action chips are interactive UI elements triggering actions tied to main content. Compact
+  and context-specific, they enhance user experience by providing additional functionality.
+  For instance, in a music app, a chip may allow users to add songs to a playlist, or share
+  them, thus seamlessly integrating extra features.
+</p>
+<div class="tds-u-flex tds-u-flex-gap-1">
+  <tds-chip size="lg" onclick="alert('Clicked chip!')">
```
Comment (alexandereneroth) [inline src/app/pages/chips-page/chips-page.component.html:9]:
```suggestion
  <tds-chip size="lg" (click)="alert('Clicked chip!')">
```
I believe that is the Angular way of making a click listener

File: src/app/pages/chips-page/chips-page.component.html [hunk 2]
```diff
@@ -1 +1,55 @@
...
+    <tds-icon slot="prefix" name="lock_inactive" size="16px"></tds-icon>
+    <span slot="label">Lock</span>
+  </tds-chip>
+  <tds-chip size="lg" onclick="alert('Clicked chip!')">
```
Comment (alexandereneroth) [inline src/app/pages/chips-page/chips-page.component.html:13]:
```suggestion
  <tds-chip size="lg" (click)="alert('Clicked chip!')">
```

File: src/app/pages/chips-page/chips-page.component.html (hunk 1 above)
Comment (mJarsater) [reply src/app/pages/chips-page/chips-page.component.html:9]: These kinds of thing I feel can just be commited straight away, no need to wait for me to get back to it.

File: src/app/pages/chips-page/chips-page.component.html (hunk 2 above)
Comment (mJarsater) [reply src/app/pages/chips-page/chips-page.component.html]: These kinds of thing I feel can just be commited straight away, no need to wait for me to get back to it.

File: src/app/pages/chips-page/chips-page.component.html (hunk 1 above)
Comment (mJarsater) [reply src/app/pages/chips-page/chips-page.component.html:9]: Sorry, no. They need to be onclick inorder to work "inline"

**Review by alexandereneroth — APPROVED** [review]
LGTM

File: src/app/pages/chips-page/chips-page.component.html (hunk 1 above)
Comment (mJarsater) [reply src/app/pages/chips-page/chips-page.component.html:9]: Yes

**Pull request merged**